src/runtime/internal/ssr.ts

```diff
@@ -6,15 +6,29 @@ export const invalid_attribute_name_character = /[\s'">/=\u{FDD0}-\u{FDEF}\u{FFF
 // https://html.spec.whatwg.org/multipage/syntax.html#attributes-2
 // https://infra.spec.whatwg.org/#noncharacter
 
-export function spread(args, classes_to_add) {
+export function spread(args, attrs_to_add) {
 	const attributes = Object.assign({}, ...args);
-	if (classes_to_add) {
-		if (attributes.class == null) {
-			attributes.class = classes_to_add;
-		} else {
-			attributes.class += ' ' + classes_to_add;
+	if (attrs_to_add) {
+		const classes_to_add = attrs_to_add.classes;
+		const styles_to_add = attrs_to_add.styles;
+
+		if (classes_to_add) {
+			if (attributes.class == null) {
```

Comment:
Strict type `===`?

Reply:
Depends, if this should catch the `undefined` case as well (I guess it should) then this should be `==`